### PR TITLE
Feat: adds automatic card background color calculation

### PR DESCRIPTION
**Pull request by eljefe223**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

By default `fluent-card` now automatically applies `neutralFillCard` to its background, which that is calculated from its parent design system provider. If a custom background color is desired the attribute `card-background-color` is now available, this will reset the cards design system to that value.

### CODE REVIEW

File: packages/web-components/src/card/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+id: card
+title: fluent-card
+sidebar_label: card
+custom_edit_url: https://github.com/microsoft/fluentui/edit/master/packages/web-components/src/card/README.md
+---
```
Comment (chrisdholt) [inline packages/web-components/src/card/README.md:6]:
```suggestion
# Fluent Card
```

File: change/@fluentui-web-components-2020-09-18-12-06-28-users-jes-card-improvements.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Feat: adds automatic card background color calculation",
```
Comment (chrisdholt) [inline change/@fluentui-web-components-2020-09-18-12-06-28-users-jes-card-improvements.json:3]:
```suggestion
"comment": "adds automatic card background color calculation based on the parent design system",
```

File: packages/web-components/src/card/index.ts [hunk 3]
```diff
@@ -47,12 +64,22 @@ export class FluentCard extends FluentDesignSystemProvider
...
+    }
+  }
+
   connectedCallback(): void {
     super.connectedCallback();
-
-    if (this.backgroundColor === undefined) {
-      this.setAttribute('use-defaults', '');
-    }
+    Observable.getNotifier(this.provider?.designSystem).subscribe(this, 'backgroundColor');
```
Comment (nicholasrice) [inline packages/web-components/src/card/index.ts:81]: I think you also want to subscribe to palette changes here

File: packages/web-components/src/card/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Card
+
+The `fluent-card` component is a visual container and design system provider. By default `fluent-card` applies `neutralFillCard` to its background that is calculated from its parent design system provider. If a custom background color is desired the attribute `card-background-color` is available, this will reset the cards design system to that value. Cards are snapshots of content that are typically used in a group to present collections of related information.
+
+## Usage
+
+```html live
```
Comment (chrisdholt) [inline packages/web-components/src/card/README.md:7]:
```suggestion
```html
```
No live needed

**Review by nicholasrice — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**